package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "devDependencies": {
```

Comment:
You'll want the following fields also:

* `name`
* `version`
* `description`
* `main`

Probably more as well...

Reply:
Thanks